d/drupal-slick_views/drupal-slick_views_ubi_8.5.sh

```diff
@@ -0,0 +1,91 @@
+#!/bin/bash -e
+
+# -----------------------------------------------------------------------------
+#
+# Package       : slick_views
+# Version       : 8.x-2.4
+# Source repo   : https://git.drupalcode.org/project/slick_views
+# Tested on     : UBI 8.5
+# Language      : PHP
+# Travis-Check  : True
+# Script License: Apache License, Version 2 or later
+# Maintainer    : Haritha Patchari<haritha.patchari@ibm.com>
+#
+# Disclaimer: This script has been tested in root mode on given
+# ==========  platform using the mentioned version of the package.
+#             It may not work as expected with newer versions of the
+#             package and/or distribution. In such case, please
+#             contact "Maintainer" of this script.
+#
+# ----------------------------------------------------------------------------
+#Variables
+
+PACKAGE_NAME=slick_views
+CORE_PACKAGE_NAME=drupal
+PACKAGE_URL=https://git.drupalcode.org/project/slick_views
+CORE_PACKAGE_URL=https://github.com/drupal/drupal
+#PACKAGE_VERSION is configurable can be passed as an argument.
+PACKAGE_VERSION=${1:-8.x-2.4}
+
+yum module enable php:7.4 -y
+yum install -y git php php-json php-dom php-mbstring zip unzip gd gd-devel php-gd php-pdo php-mysqlnd
+php -r "copy('https://getcomposer.org/installer', 'composer-setup.php');" && php composer-setup.php --install-dir=/bin --filename=composer
+OS_NAME=`cat /etc/os-release | grep PRETTY_NAME | cut -d '=' -f2 | tr -d '"'`
+#Check if package exists
+if [ -d "$CORE_PACKAGE_NAME" ] ; then
+  rm -rf $CORE_PACKAGE_NAME
+  echo "$CORE_PACKAGE_NAME  | $PACKAGE_VERSION | $OS_NAME | GitHub | Removed existing package if any"
+
+fi
+if ! git clone $CORE_PACKAGE_URL $CORE_PACKAGE_NAME; then
+        echo "------------------$PACKAGE_NAME:clone_fails---------------------------------------"
+                echo "$CORE_PACKAGE_URL $CORE_PACKAGE_NAME"
```

Comment:
```suggestion
            echo "$CORE_PACKAGE_URL $CORE_PACKAGE_NAME"
```
keep one uniform indentation